python/mxnet/libinfo.py

```diff
@@ -73,6 +73,11 @@ def find_lib_path(prefix='libmxnet'):
                            'List of candidates:\n' + str('\n'.join(dll_path)))
     if os.name == 'nt':
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
+        if sys.version_info >= (3, 8):
```

Comment:
It's more pythonic to use try-except syntax and just attempt to call add_dll_directory
It will ensure compatibility with interpreters different to cpython

Reply:
Check the version here because only 3.8 or above needs to call this

Reply:
It's nevertheless bad practice as there are other python interpreters with different versioning schemes